src/Options.jl

```diff
@@ -22,8 +24,8 @@ end
   All types for this purpose have this abstract type as super-type.
   
   Required fields are: `name`, `lastchanged`, `options`
-  """
-abstract AbstractOptionsODE <: Any
+"""
+AbstractOptionsODE
```

Comment:
Do you think it's better to add Compat? (it's not used by this package currently)